[2024-03-06T15:01:07.206-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_crypto_dag.AVAX scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-03-06T15:01:07.219-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_crypto_dag.AVAX scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-03-06T15:01:07.219-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T15:01:07.239-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): AVAX> on 2024-01-03 00:00:00+00:00
[2024-03-06T15:01:07.248-0300] {standard_task_runner.py:60} INFO - Started process 91605 to run task
[2024-03-06T15:01:07.251-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_crypto_dag', 'AVAX', 'scheduled__2024-01-03T00:00:00+00:00', '--job-id', '328', '--raw', '--subdir', 'DAGS_FOLDER/get_crypto.py', '--cfg-path', '/tmp/tmpjopfd3ox']
[2024-03-06T15:01:07.252-0300] {standard_task_runner.py:88} INFO - Job 328: Subtask AVAX
[2024-03-06T15:01:07.320-0300] {task_command.py:423} INFO - Running <TaskInstance: get_crypto_dag.AVAX scheduled__2024-01-03T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T15:01:07.425-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_crypto_dag' AIRFLOW_CTX_TASK_ID='AVAX' AIRFLOW_CTX_EXECUTION_DATE='2024-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-03T00:00:00+00:00'
[2024-03-06T15:01:20.618-0300] {history.py:90} ERROR - AVAX: No timezone found, symbol may be delisted
[2024-03-06T15:01:30.622-0300] {python.py:202} INFO - Done. Returned value was: None
[2024-03-06T15:01:30.639-0300] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=get_crypto_dag, task_id=AVAX, execution_date=20240103T000000, start_date=20240306T180107, end_date=20240306T180130
[2024-03-06T15:01:30.668-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T15:01:30.690-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
